src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -490,10 +490,12 @@ public static partial class Marshal
         public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
         [System.Security.SecurityCriticalAttribute]
         public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+#if NETCOREAPP11
```

Comment:
Hmm, I hadn't seen that: https://github.com/dotnet/corefx/search?l=csharp&q=netnative&type=Code&utf8=%E2%9C%93

I can change this though if that's what you'd prefer.